dev-util/clazy/clazy-1.4.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A plugin which allows clang to understand Qt semantics"
+DESCRIPTION="clazy is a compiler plugin which allows clang to understand Qt \
+semantics. You get more than 50 Qt related compiler warnings, ranging from \
+unneeded memory allocations to misusage of API, including fix-its for \
+automatic refactoring."
+HOMEPAGE="https://www.kdab.com"
+COPYRIGHT="2016-2018 Sérgio Martins"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/clazy/1.4/src/clazy-1.4.tar.xz"
+CHECKSUM_SHA256="3f5d5e148c9e9c4e43f095796261794da5385578d2375b12c9179d340d6d5a8a"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	clazy$secondaryArchSuffix = $portVersion
+	cmd:clazy
+	cmd:clazy_standalone
+	lib:ClangLazy = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libLLVM_5.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libLLVM_5.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:clang
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:llvm_config
```

Comment:
the required version for cmd:llvm_config should be added, otherwise it could end being different from BUILD_REQUIRES one.

Reply:
checking, "cmd:llvm_config >= 5.0" should do it I guess?